src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let res = sourceString.split(';');
+
+  res = res.map((el) => el.split(':'));
```

Comment:
Ensure that the input string is correctly formatted. If the string does not contain the expected delimiter ':', the resulting object might not be as intended.